### ISSUE TITLE
Toggle service issue

### ISSUE THREAD

**Issue by sunnyhick** [Fate-Grand-Automata/FGA#288]
Hello, I just installed the latest release v0.20.0 but I can't seem to turn on toggle service.

I had accessibility turned off, but I turned it on after the app prompted me to. Then I tried turning it on and off, it still doesn't work and shows me the following.

![Screenshot_2020-08-23-02-03-26-161_com mathewsachin fategrandautomata](https://user-images.githubusercontent.com/70075026/90964747-484ef800-e4e5-11ea-8751-1484996f93c8.jpg)

I tried restarting and even enabled no restrictions on battery optimization. It gives me the same results. I tried deleting cache but nothing really changes. Is there anything else I should do? Or am I doing something wrong.

Device Info:
 - Device model: Xiaomi Redmi Note 9S
 - Screen size: 1080x2400
 - Android Version: Android 10
 - RAM: 6GB
